index.js

```diff
@@ -1,18 +1,59 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = '#228B22'
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+let travelerProfile = document.querySelector('div#profile')
+
+let travelerImg = travelerProfile.querySelector('img')
+travelerImg.src = traveler.photo
+travelerImg.alt = traveler.name
+
+let travelerName = travelerProfile.querySelector('h2')
+travelerName.textContent = traveler.name
+
+let travelerNickName = travelerProfile.querySelector('em')
+travelerNickName.textContent = traveler.nickname
+
 
+console.log('This is what the traveler object looks like: ', traveler)
 
 /***** Deliverable 4 *****/
+let animalSightingList = document.querySelector('ul#animals')
 
+let sightings = traveler.animalSightings
 
+// let animalLi = document.createElement('li')
+// let animalDescription = document.createElement('p')
+// let animalPhoto = document.createElement('img')
+// let animalLink = document.createElement('a')
 
+// animalLi.setAttribute('id', 'aSighting')
+
+sightings.forEach(function(sighting) {
+    let animalLi = document.createElement('li')
+    let animalDescription = document.createElement('p')
+    let animalPhoto = document.createElement('img')
+    let animalLink = document.createElement('a')
+
+    animalLi.setAttribute('data-id', sighting.id)
```

Comment:
For dataset attributes, you can also use this syntax:

```js
li.dataset.id = sighting.id
```

This will add a `data-id` attribute to the element, like this:

```html
<li data-id="1">...</li>
```